css/properties/-webkit-line-grid.json

```
@@ -5,7 +5,8 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "17",
+              "version_removed": "35"
```

Added in Chrome 17 (https://github.com/caugner/css-properties/commit/ec6146111b20d422fdc04adfde535bbe8b967050), but removed in Chrome 34 (https://github.com/caugner/css-properties/commit/4560f7fda4a8b2300fc5c9135085a3ed7b7651ee).

```suggestion
              "version_added": "17",
              "version_removed": "34"
```